composer.json

```
@@ -30,12 +30,12 @@
         "phpstan/phpstan-symfony": "^1.0",
         "phpunit/phpunit": "^8.0",
         "php-cs-fixer/shim": "^3.0",
-        "symfony/browser-kit": "^4.3 || ^5.0 || ^6.0",
-        "symfony/dotenv": "^4.3 || ^5.0 || ^6.0",
+        "symfony/browser-kit": "^4.3 || ^5.0 || ^6.0 || ^6.0",
+        "symfony/dotenv": "^4.3 || ^5.0 || ^6.0 || ^6.0",
         "symfony/monolog-bundle": "^3.1",
         "jackalope/jackalope-doctrine-dbal": "^1.3.4",
```

this needs also increased:

```suggestion
        "jackalope/jackalope-doctrine-dbal": "^1.3.4 || ^2.0",
```